alb-shared-to-k8s.tf

```diff
@@ -175,3 +175,14 @@ resource "aws_security_group_rule" "batcave_alb_shared_egress" {
   cidr_blocks       = ["0.0.0.0/0"]
   security_group_id = aws_security_group.batcave_alb_shared[0].id
 }
+
+data "aws_wafv2_web_acl" "cms_waf" {
+  name  = "RegSamQuickACLEnforcingV2"
+  scope = "REGIONAL"
+}
```

Comment:
This is redundant with the one created in the other file; all .tf files exist in the same context.